src/shared/components/layout/LayoutSplit.js

```diff
@@ -5,10 +5,10 @@ import styles from './LayoutSplit.module.css';
 
 class LayoutSplit extends Component {
     render() {
-        const { splitAt, left, right, className, ...props } = this.props;
+        const { splitAt, left, right, className, activeRef, ...props } = this.props;
```

Comment:
The introduction of `activeRef` is strange. You can always do `findDOMNode` to get the node element.